src/runtime/Makefile

```
@@ -612,13 +612,32 @@ go-test: $(GENERATED_FILES)
 	go test -v -mod=vendor ./...
 
 fast-test: $(GENERATED_FILES)
+	go clean -testcache
 	for s in $$(go list ./...); do if ! go test -failfast -v -mod=vendor -p 1 $$s; then break; fi; done
 
+GOLANG_LINT_FILE := ../../../tests/.ci/.golangci.yml
+GOLANGCI_LINT_CMD := $(shell command -v golangci-lint 2> /dev/null)
```

I'd be tempted to create a variable for the name of the linter as that appears in messages below:

```suggestion
GOLANGCI_LINT_NAME = golangci-lint
GOLANGCI_LINT_CMD := $(shell command -v $(GOLANGCI_LINT_NAME) 2>/dev/null)
```

---

@jodh-intel updated as your suggestion.